### PR TITLE
Add HDFS version requirement to active sync docs

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
@ZacBlanco PTAL at this doc change related to #9609

File: docs/en/advanced/Namespace-Management.md [hunk 1]
```diff
@@ -288,7 +288,9 @@ hello
 
 ### Metadata Active Sync for HDFS
 In version 2.0, we introduced a new feature for maintaining synchronization between Alluxio space and the UFS when the UFS is HDFS.
-The feature, called active sync, listens for HDFS events and periodically synchronizes the metadata between the UFS and Alluxio namespace as a background task on the master.
+The feature, called active sync, listens for HDFS events and periodically synchronizes the metadata between the UFS and Alluxio namespace as a background task on the master. 
+Because active sync feature depends on HDFS events, this feature is only available when the UFS HDFS versions is later than 2.6.1.
+Be sure to specify a value for `alluxio.underfs.version` in your configuration file. It defaults to 2.2, which does not support HDFS events.
```
Comment (ZacBlanco) [inline docs/en/advanced/Namespace-Management.md]: There is a page somewhere with this property that references available versions. Maybe link to that?

Comment (ZacBlanco) [reply docs/en/advanced/Namespace-Management.md]: https://docs.alluxio.io/os/user/stable/en/ufs/HDFS.html?q=alluxio.underfs.version#supported-hdfs-versions

File: docs/en/advanced/Namespace-Management.md [hunk 2]
```diff
@@ -288,7 +288,10 @@ hello
...
+Please refer to [HDFS Under Store]({{ '/en/ufs/HDFS.html?#supported-hdfs-versions' | relativize_url }}) for a list of supported Hdfs versions.
```
Comment (ZacBlanco) [inline docs/en/advanced/Namespace-Management.md]: still have a hanging `?`. It shouldn't affect the user's experience, but probably better to clean up the link.

Comment (yuzhu) [reply docs/en/advanced/Namespace-Management.md]: fixed

**Comment by yuzhu** [comment]
@ZacBlanco fixed the link, PTAL. thanks

**Comment by yuzhu** [comment]
@apc999 please take a look at this doc change and merge when you are ok with it. thanks.

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please

**Pull request merged**